src/main/java/core/basesyntax/CsvFileReader.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReader {
+    public List<Operation> readFile(String filepath) {
+        List<Operation> operationList = new ArrayList<>();
+        try {
+            FileReader fileReader = new FileReader(filepath);
+            if (!filepath.toLowerCase().endsWith(".csv")) {
+                throw new RuntimeException("File is not in .csv format");
+            }
+            CSVReader csvReader = new CSVReader(fileReader);
+            List<String[]> raws = csvReader.readAll();
+            if (raws.get(0)[0].equals("type")) {
+                raws.remove(0);
+            }
```

Comment:
FYI: 
you can skip header with CSVReader configuration, this one of options :
CSVReader csvReader = new CSVReaderBuilder(fileReader)
                .withSkipLines(1)
                .build();

Reply:
yes, but what if we recieve input file without header?
this checking was made to work with both types of input files - with header and without

Reply:
it's a best practice to include header into csv file, cause it allows the file to be self-documenting, and your solution won't work if header has different column sequence and type won't be on 0 index

Reply:
fixed